packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -198,6 +215,8 @@ hub.post("/", async (req, res) => {
         configObject[botName]?.environmentVariables?.STORE_MULTI_CHAIN_BLOCK_NUMBERS
       );
       botConfigs[botName] = botConfig;
+      // gets a spoke url based on execution size or fallback to default spoke url if non specified
+      const spokeUrl = getSpokeUrl(botConfig.spokeExecutionSize);
```

Comment:
nit: this may be used for more than just size.

Maybe we should name it something that's more agnostic to the property that we care about. For instance, `spokeName` might be more preferred over `spokeExecutionSize`.

Reply:
i agree, was thinking the same thing

Reply:
renamed this to `spokeLabel`

Reply:
actually changing this to spokeUrlName because it conflicts with another usage of spokeName, its also more specific